### PR TITLE
Implementation transfer isi to Iroha_CLI

### PR DESCRIPTION
**Pull request by stepanLav**
### Description of the Change
Implementation transfer isi for iroha_CLI

### Benefits
Possible to use "Transfer" isi with iroha_CLI

### CODE REVIEW

**Review by humb1t — APPROVED** [review]

File: iroha_client_cli/src/main.rs [hunk 1]
```diff
@@ -216,18 +219,25 @@ mod asset {
...
-                .arg(Arg::with_name(QUANTITY).long(QUANTITY).value_name(QUANTITY).help("Asset's quantity as a number.").takes_value(true).required(true))
-                )
+                    .about("Use this command to Mint Asset in existing Iroha Account.")
+                    .arg(Arg::with_name(ASSET_ACCOUNT_ID).long(ASSET_ACCOUNT_ID).value_name(ASSET_ACCOUNT_ID).help("Account's id as double-quoted string in the following format `account_name@domain_name`.").takes_value(true).required(true))
+                    .arg(Arg::with_name(ASSET_ID).long(ASSET_ID).value_name(ASSET_ID).help("Asset's id as double-quoted string in the following format `asset_name#domain_name`.").takes_value(true).required(true))
+                    .arg(Arg::with_name(QUANTITY).long(QUANTITY).value_name(QUANTITY).help("Asset's quantity as a number.").takes_value(true).required(true))
+            )
+            .subcommand(
+                App::new(TRANSFER)
+                    .about("Use this command to Mint Asset in existing Iroha Account.")
```
Comment (eadventurous) [inline iroha_client_cli/src/main.rs:229]: `Use this command to Transfer Assets...`

**Review by eadventurous — CHANGES_REQUESTED** [review]

**Comment by humb1t** [comment]
@stepanLav please read about our git flow and squash all your commits into one (see CONTRIBUTING.md)

File: iroha_client_cli/src/main.rs (hunk 1 above)
Comment (stepanLav) [reply iroha_client_cli/src/main.rs]: I fixed it, thank you.

**Comment by stepanLav** [comment]
@humb1t 
I got it, thank you, I got valuable experience.)

**Comment by eadventurous** [comment]
@stepanLav please squash the commits one last time, and we will merge)

**Comment by stepanLav** [comment]
@eadventurous 
I did it, you can merge

**Review by eadventurous — APPROVED** [review]

**Pull request merged**